### DISCUSSION TITLE
unable to acess student benifits

### DISCUSSION THREAD

**Discussion by sleepy5** [community/community#135407] [GitHub Education] [answered]
### Hi everyone,

My request was approved on Aug 6, but I haven't received education benefits.

**Comment by davevad93** [comment, marked as answer]
Hi @sleepy5  ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sleepy5 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by sleepy5** [reply]
ok, thank you.
